src/goto-programs/skolemization.cpp

```diff
@@ -0,0 +1,69 @@
+/// \file skolemization.h
+/// Rename variables in existentially quantified statements.
+/// Author: Diffblue Ltd.
+
+#include "skolemization.h"
+
+#include "goto_model.h"
+
+#include <ansi-c/expr2c.h>
+
+#include <util/expr_util.h>
```

Comment:
:pick: Includes of `<ansi-c/expr2c.h>` and `<util/expr_util.h>` appear to be unused.

Reply:
Ok, this is now resolved.